bucketstracking_agentmanager.go

```diff
@@ -91,12 +91,21 @@ func CreateBucketsTrackingAgentManager(ctx context.Context, opts BucketsTracking
 		zap.Any("bootstrapConfig", bootstrapConfig),
 		zap.String("networkType", networkType))
 
+	createAgentTimeout := opts.CreateAgentTimeout
+	if createAgentTimeout == 0 {
+		createAgentTimeout = 7 * time.Second
```

Comment:
Why 7 seconds?

Reply:
7 seconds is what we typically use as the connection timeout in the SDK, I'm unsure where that value originated.